src/components/CurrentWeather.vue

```diff
@@ -50,8 +43,10 @@
 <script>
 export default {
```

Comment:
Consider using functional component

Reply:
@probil, what is the reason of using functional components?

Reply:
The same reason as function components in react. They are cheaper to render. They are lightweight. They are just functions (converted to function under the hood).